fideslib/db/session.py

```diff
@@ -13,8 +13,21 @@
 logger = logging.getLogger(__name__)
 
 
-def get_db_engine(database_uri: str | URL) -> Engine:
-    """Return a database engine."""
+def get_db_engine(
+    config: FidesConfig,
+    database_uri: str | URL | None = None,
+) -> Engine:
```

Comment:
I don't know if config should be required here since you prioritize the `database_uri` field if it exists and bypass the config altogether. I might throw an exception if neither one is passed in to tell the developer that one is needed.